x-pack/plugins/index_management/public/sections/index_list/components/index_table/index_table.js

```diff
@@ -336,7 +335,13 @@ export class IndexTableUi extends Component {
       indices,
       intl,
       loadIndices,
+      indicesLoading
     } = this.props;
+    const emptyState = indicesLoading ? (
```

Comment:
I think we should hide the search bar while `indicesLoading` too, since there's no point in the user interacting with it or with the filters when the table isn't visible.

![image](https://user-images.githubusercontent.com/1238659/50021449-02126900-ff8e-11e8-9c44-14ddc7d2ad38.png)